ibis/expr/types/relations.py

```diff
@@ -124,7 +125,7 @@ def __rich_console__(self, console, options):
         from ibis.expr.types.pretty import to_rich_table
 
         if not ibis.options.interactive:
-            return console.render(Text(self._repr()), options=options)
+            return Text(self._repr())
```

Comment:
Suspect this will break. This method should return an iterable.

Reply:
When I return `[Text(self._repr())]` the double newline appears again.